sys/riscv/include/asm.h

```diff
@@ -47,8 +47,8 @@
 #define	_C_LABEL(x)	x
 
 #define	ENTRY(sym)						\
-	.text; .globl sym; .type sym,@function; .align 4; sym:
-#define	END(sym) .size sym, . - sym
+	.text; .globl sym; .type sym,@function; .align 4; sym: .cfi_startproc;
+#define	END(sym) .size sym, . - sym;  .cfi_endproc;
```

Comment:
Tiny nit, I would probably put `.cfi_endproc` first so it is "inside" the normal end.  This is how it is usually done when it is hand-written (or at least when I've hand-written them).

Reply:
Makes sense, will update.